accounts/views.py

```diff
@@ -222,8 +222,14 @@ def resend_activation(request):
     if request.method == 'POST':
         form = ReactivationForm(request.POST)
         if form.is_valid():
-            user = form.cleaned_data['user']
-            send_activation(user)
+            username_or_email = form.cleaned_data['user']
```

Comment:
So now activation works with usernames and emails right? This is a new feature not explained in the PR or issue description right?

Reply:
The clean_user method of the form used to check email and username, now this is made in the view.